models/modules/diffusion_generator.py

```diff
@@ -128,6 +128,13 @@ def restoration_ddpm(
         ref,
     ):
         phase = "test"
+        frame = 0
+        if len(y_cond.shape) == 5:
+            frame = y_cond.shape[1]
```

Comment:
Please add a comment here to explain what is being done.

Reply:
comment is added. the objective is to align vid process with the standard DDPM process